spec/unit/owner_spec.rb

```diff
@@ -2,11 +2,9 @@
 
 module Pod::TrunkApp
   describe Owner do
-    it 'creates an owner' do
+    it 'initializes an owner' do
       owner = nil
-      lambda do
-        owner = Owner.find_or_create_by_email_and_update_name(' appie@example.com ', ' Appie ')
-      end.should.change { Owner.count }
+      owner = Owner.find_or_initialize_by_email_and_name(' appie@example.com ', ' Appie ')
```

Comment:
That's why I changed the tests + methods name to `find_or_initialize_by_email_and_name`.